console/src/main/java/org/matrix/console/services/EventStreamService.java

```diff
@@ -641,10 +635,15 @@ private Notification buildNotification() {
 
         PendingIntent pi = PendingIntent.getActivity(this, 0, i, 0);
 
-        notification.setLatestEventInfo(this, getString(R.string.app_name),
-                "Listening for events",
-                pi);
-        notification.flags |= Notification.FLAG_NO_CLEAR;
```

Comment:
In order to build with SDK 23, I had to use [Notification.Builder](https://developer.android.com/reference/android/app/Notification.Builder.html) rather than using [Notification](https://developer.android.com/reference/android/app/Notification.html) directly.
Notification.Builder has an equivalent method [setOngoing](https://developer.android.com/reference/android/app/Notification.Builder.html#setOngoing(boolean))(boolean ongoing) doing the same thing I'm calling one line below.
Hopefully I haven't interpreted it wrong.
Thanks for the review anyway.